docs/src/enterprise/getstarted/migrate-from-fluffy.md

```diff
@@ -0,0 +1,147 @@
+
+
+### Authentication configuration
+
+Most Fluffy `auth.*` settings migrate directly to lakeFS Enterprise with the same structure. Below are the differences between the configurations.
+
+
+!!! note "SAML"
+    
+    === "Fluffy (old)"
+        
+        ```yaml
+        # fluffy.yaml
+        auth:
+          post_login_redirect_url: http://localhost:8000/
+          saml:
+            enabled: true 
+            external_user_id_claim_name: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name
+        ```
+    
+    === "lakeFS Enterprise (new)"
+     
+        ```yaml
+        # lakefs.yaml
+        auth:
+          cookie_auth_verification:
+            external_user_id_claim_name: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name # This field was moved here!
+          providers:
+            saml:
+			  post_login_redirect_url: http://localhost:8000/ # This field was moved here!
+              # enabled: true  // This field was dropped! 
+        ```
+
+
+!!! note "OIDC"
+   
+    === "Fluffy (old)"
+        
+        ```yaml
+        # fluffy.yaml
+        auth:
+          post_login_redirect_url: http://localhost:8000/
+          oidc:
+            enabled: true
+        ```
+    
+    === "lakeFS Enterprise (new)"
+        
+        ```yaml
+        # lakefs.yaml
+        auth:
+          oidc:
+            post_login_redirect_url: http://localhost:8000/ # This field was moved here!
+            # enabled: true  // This field was dropped! 
+        ```
+
+
+!!! note "LDAP"
+   
+    === "Fluffy (old)"
+        
+        ```yaml
+        # fluffy.yaml
+        auth:
+          ldap:
+        ```
+    
+    === "lakeFS Enterprise (new)"
+        
+        ```yaml
+        # lakefs.yaml
+        auth:
+		  ldap:
+            default_user_group: "admins" // This field moved here!
+        ```
+
+!!! note "AWS"
+    The structure of AWS STS authentication remains the same.
+
+### Authorization configuration
+
+!!! note "RBAC"
+   
+    === "Fluffy (old)"
+        
+        ```yaml
+        # fluffy.yaml
+        auth:
+          serve_disable_authentication: false
+          serve_listen_address: "localhost:8000"
+		  cache:
+			enabled: true
+        ```
+    
+    === "lakeFS Enterprise (new)"
+        
+        ```yaml
+        # lakefs.yaml
+        auth:
+          # serve_disable_authentication: false // this field was dropped!
+          # serve_listen_address: "localhost:8000" // this field was dropped!
+		  cache:
+			enabled: true
+        ```
+
+### Database Configuration
+
+Use the same database configuration, but only in the lakeFS Enterprise config:
+
+```yaml
+database:
+  type: postgres
+  postgres:
+    connection_string: "postgres://user:pass@host:5432/db?sslmode=disable"
+```
+
+### License Configuration 
+
+lakeFS Enterprise requires a license to run. 
+
+```yaml
+license:
+  # use one of the two below. Using both results in an error.
+  contents: <enterprise license>
+  path: <path to license>
+```
```

Comment:
IMO that section should be modified. 
- License is not related to fluffy-eol scope. 
- At the same time it is for specific case - users who migrate from fluffy might be using `lakefs` OSS binary. 
So add this to pre-requisites at the top, part of having: 
1. lakeFS license 
2. lakeFS-Enterprise image or binary 
3. Minimum version for lakeFS to use